beta-src/src/App.tsx

```diff
@@ -2,14 +2,11 @@ import * as React from "react";
 import "./assets/css/App.css";
 import Box from "@mui/material/Box";
 import WDMain from "./components/ui/WDMain";
-import WDScrollButton from "./components/ui/WDScrollButton";
-import { ScrollButtonState } from "./enums/UIState";
 
 const App: React.FC = function (): React.ReactElement {
   return (
     <Box className="App">
-      {/* <WDMain /> */}
-      <WDScrollButton direction={ScrollButtonState.BACK} />
+      <WDMain />
```

Comment:
i think you need to reset this file to the head? we should not be making any changes in App for this PR

git checkout HEAD -- my-file.txt